src/client/Root.js

```diff
@@ -1,24 +1,22 @@
-import React, {Component} from 'react';
```

Comment:
Ohhhh! didn't see the change to stateless functions. nice!
@jordanh per dan abramov's medium post, i think the winds are changing to using webpack's stock HMR solution instead of proxying all of the react component to achieve HMR. This will be especially true since detecting stateless components in babel is error prone. Just something to put on the radar when we start focusing on the front end build.